src/modules/tas_studio/mod.rs

```diff
@@ -1602,6 +1755,12 @@ pub unsafe fn on_tas_playback_frame(
             strafe_state.prev_frame_input.yaw = view_angles[1].to_radians();
         }
 
+        // Store the rendered viewangles
+        // Viewangles is always available. Same as origin.
+        // Vieworigin is something that is not available per player. It is calculated upon
+        // rendering.
```

Comment:
```suggestion
        // Store the rendered viewangles.
        // Viewangles are always available. Same as origin.
        // Vieworigin is something that is not available per player. It is calculated upon
        // rendering.
```
Also what do you mean by "same as origin" and also "per player"?

Reply:
I don't really know why I wrote these. Deleeted